backend/oasst_backend/exceptions.py

```diff
@@ -17,6 +17,7 @@ class OasstErrorCode(IntEnum):
     GENERIC_ERROR = 0
     DATABASE_URI_NOT_SET = 1
     API_CLIENT_NOT_AUTHORIZED = 2
+    SERVER_ERROR = 3
```

Comment:
;-) that one is not much different from GENERIC_ERRER

Reply:
In my opinion GENERIC_ERROR is an error we do not understand, while SERVER_ERROR is error due to our fault.